src/inim.nim

```diff
@@ -32,13 +41,34 @@ const
   ConfigDir = getConfigDir() / "inim"
   RcFilePath = ConfigDir / "inim.ini"
 
+proc getOrSetSectionKeyValue(dict: var Config, section, key,
+    default: string): string =
+  ## Get a value or set default for that key
+  ## This is for when users have older versions of the config where they may be missing keys
+  result = dict.getSectionValue(section, key)
+  if result == "":
+    #Option not present, we should set instead of erroring
+    dict.setSectionKey(section, key, default)
+    result = default
+
+proc loadRCFileConfig(path: string): Config =
+  # Perform any config migrations
+  result = loadConfig(path)
+  # Refresh config to use showColor instead of ShowColor (consistency)
+  if result.getSectionValue("Style", "ShowColor") != "":
+    result.setSectionKey("Style", "showColor", result.getSectionValue("Style", "ShowColor"))
+    result.delSectionKey("Style", "ShowColor")
+  result.writeConfig(path)
+
```

Comment:
what's happening here?

Reply:
One of the config options was in an inconsistent case (ie ShowColor). Everything else uses camelCase, this essentially migrates any existing config over to the new key (showColor)

Reply:
I got rid of this. Turns out the rcfile didn't work in the first place, so just changing it should be fine. Look at this PR instead to fix this https://github.com/inim-repl/INim/pull/91